### PR TITLE
Update kube-inject version to use minor version

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

**Comment by sebastienvas** [comment]
@ayj Does the manager e2e test use the default docker image (ie in my change 0.1)?

**Comment by sebastienvas** [comment]
jenkins build manager/presubmit

**Comment by ayj** [comment]
The manager's e2e integration presubmit test uses the images built from the PR under test.

File: Jenkinsfile [hunk 1]
```diff
@@ -14,9 +14,18 @@ def bazel = new Bazel()
 // This should be updated for a release branch.
 ISTIO_VERSION_URL = 'https://raw.githubusercontent.com/istio/istio/master/istio.RELEASE'
 
+
+def setVersions() {
+  def version = sh(returnStdout: true, script: "curl ${ISTIO_VERSION_URL}").trim()
+  def v = version.tokenize('.')
+  env.ISTIO_VERSION = version
+  env.ISTIO_MINOR_VERSION = "${v[0]}.${v[1]}"
```
Comment (ayj) [inline Jenkinsfile]: This is the latest non-patch version of the current release cycle, correct? For example, given x.y.z, ISTIO_MINOR_VERSION would always be x.y?

Comment (sebastienvas) [inline Jenkinsfile]: yes.

**Pull request merged**